app/src/main/java/org/thoughtcrime/securesms/conversation/start/NewConversationHomeFragment.kt

```diff
@@ -48,16 +48,14 @@ class NewConversationHomeFragment : Fragment() {
     @Inject
     lateinit var textSecurePreferences: TextSecurePreferences
 
-    var delegate = MutableStateFlow<NewConversationDelegate?>(null)
+    var delegate = MutableStateFlow<NewConversationDelegate>(NullNewConversationDelegate)
 
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View = createThemedComposeView {
-        // Warning, brittle code, NewConversationScreen will not be visible if no delegate is set
-        // before onCreateView is called.
-        delegate.collectAsState().value?.let {
+        delegate.collectAsState().value.let {
```

Comment:
Do you still need a let without any new values?

Reply:
inlined.